src/client.rs

```diff
@@ -99,6 +102,7 @@ impl Client {
     where
         F: Future<Item = (), Error = ()> + Send + 'static,
     {
-        Executor::new(self.channel.cq()).spawn(f)
+        let kicker = self.kicker.clone();
+        Executor::new(self.channel.cq()).spawn(f, kicker)
```

Comment:
What if Client is dropped before `f` is resolved?

Reply:
`Kicker::clone` bumps call's ref count.

Reply:
Any tests to cover it?